### ISSUE TITLE
transition of v5.1 to production

### ISSUE THREAD

**Issue by funderburkjim** [funderburkjim/kosha-dev#23]
After recent work (issues 20-22),  I think the 5.1 dev version is ready to join the main Cologne displays.
@drdhaval2785,  if you agree, then I'll begin thinking about how to accomplish this transition.
Do you have any preliminary thoughts on this transition?

**Comment by drdhaval2785** [comment]
I completely agree. I do not have any thoughts on the integration, other than the fact that I am very happy. 

If users give some feedback, we will anyhow make necessary corrections.

**Comment by funderburkjim** [comment]
## production installed.

Everything looks ok to me both on local server and Cologne server.

The only TODO items that I am aware of are
* csl-homepage
* Worldcat link

@drdhaval2785  See [v5.1/convert/readme](https://github.com/funderburkjim/kosha-dev/tree/main/v5.1/convert) for notes regarding the conversion from dev to production.

----------------------------------------------------------
to make local version,
* cd /c/xampp/htdocs/cologne
*  pull the following repositories from sanskrit-lexicon organization at Github; 
   * csl-orig, csl-pywork, csl-websanlexicon, csl-apidev. Optionally, hwnorm1.
* cd csl-pywork/v02
* sh generate_dict.sh abch  ../../abch

Now the local displays will be at urls:
* http://localhost/cologne/abch/web/
* http://localhost/cologne/simple/

I have done the similar things on Cologne server, so the Cologne displays are at the usual urls
* https://sanskrit-lexicon.uni-koeln.de/scans/ABCHScan/2023/web/index.php
* https://sanskrit-lexicon.uni-koeln.de/simple/

**Comment by drdhaval2785** [comment]
Great

**Comment by drdhaval2785** [comment]
csl-homepage is yet pending to be updated @funderburkjim ?
Once it is done, the world at large will be able to look at this dictionary.

**Comment by funderburkjim** [comment]
@drdhaval2785 

csl-homepage updated.  readme_update.txt describes what was done.
update_version.sh revised.
homepage revised locally and on Cologne server.

Please review and correct as needed.

Would you add to csl-newsletter?

**Comment by drdhaval2785** [comment]
I will add to csl-newsletter.

**Comment by drdhaval2785** [comment]
Added to csl-newsletter vide above commit.
